.github/workflows/release-packages.yaml

```diff
@@ -223,7 +223,7 @@ jobs:
       - name: Set Image Tag
         run: echo "IMAGE_TAG=diffblue/cbmc:${{ steps.get_release_info.outputs.tag_name }}" >> $GITHUB_ENV
       - name: Build docker image
-        run: docker build -t $IMAGE_TAG .
+        run: docker build -t "$IMAGE_TAG" .
```

Comment:
🚫 What's it explaining about right now? What's the value of `$IMAGE_TAG` here? Image tags should be easy to reference by developers so shouldn't contain things like spaces or other weirdness

Reply:
I think that might have been a mistake from my part as well when I made the release. Because of the release version action we are using, and the release names I chose which were kind of elaborate, like `cbmc-5.21.2-exp`, it might be using the whole tag as a name.

Reply:
I mean the `" "` should certainly help in any case, but part of it was the release name I chose made the tag name more convoluted than it is allowed to be.